x-pack/plugins/security_solution/server/lib/timeline/saved_object/pinned_events/index.ts

```diff
@@ -215,23 +173,110 @@ export const persistPinnedEventOnTimeline = async (
   }
 };
 
+const getValidTimelineIdAndVersion = async (
```

Comment:
Let me know if you have a better idea for the name here.

Reply:
name's fine